CHANGELOG.adoc

```diff
@@ -0,0 +1,223 @@
+= Revision History
+:toc:
```

Comment:
TBH, I am not even sure whether this file should be converted at all. This is not really documentation per se. Keep this file format as simple as possible.